Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -441,6 +442,23 @@ function vtkWidgetManager(publicAPI, model) {
 
   publicAPI.updateSelectionFromXY = (x, y) => {
     if (model.pickingEnabled) {
+      // First pick SVG representation
+      for (let i = 0; i < model.widgets.length; ++i) {
+        const widget = model.widgets[i];
+        const hoveredSVGReps = widget
+          .getRepresentations()
+          .filter((r) => r.isA('vtkSVGRepresentation') && r.getHover() != null);
```

Comment:
Linter will likely complain about `!=`. Use `!==`.

Reply:
linter does not complain.

I thought it was fine to use `==`/`!=` when comparing to `null`. That way it works for both `undefined` and `null`.

I can do `!==` if you prefer though. your call.

Reply:
Ah, if you want to cover both `undefined` and `null` (and just those two) as being falsy, then yes you can use the non-strict version.